ngtools/utils.py

```diff
@@ -62,3 +65,25 @@ class bright:
     okgreen = fg.bright.green
     warning = fg.bright.yellow
     fail = fg.bright.red
+
+
+def neuroglancer_state_to_neuroglancer_url(
+        state,
+        base_url="https://neuroglancer.lincbrain.org/cloudfront/frontend/index.html"
+):
+    """
+    Takes the current State of the Neuroglancer Viewer and converts it into a url-encoded value for the browser
+
+    """
+    ordered_dict = state._json_data
+    # Convert the OrderedDict to a regular dictionary
+    dict_data = json.loads(json.dumps(ordered_dict))
+
+    # Convert the dictionary to a JSON string
+    json_str = json.dumps(dict_data)
```

Comment:
What's the advantage of doing `dumps(loads(dumps(state)))` over simply `dumps(state)`?

Reply:
Updated -- simplified things similar to https://github.com/neuroscales/ngtools/pull/1#discussion_r1553883925